doc/general.rst

```diff
@@ -368,6 +369,48 @@ Example:
    Bit-flips for row   132: 12
    Bit-flips for row   134: 3
 
+Row mapping:
+
+Map row pairs from row 3 (``--start-row``) to row 60 (``--nrows``) where the next pair is 5 rows away (``--row-jump``) from the previous one:
```

Comment:
Maybe we should specify that `nrows` sets the total number of rows, but the rows are counted from 0 to `nrows`-1:
```suggestion
Map row pairs from row 3 (``--start-row``) to row 59 (``--nrows`` - 1, as we count total number of rows from 0) where the next pair is 5 rows away (``--row-jump``) from the previous one:
```